geoh5py/objects/drape_model.py

```diff
@@ -88,11 +99,21 @@ def centroids(self):
         return self._centroids
 
     @property
-    def layers(self) -> np.ndarray | None:
+    def extent(self) -> np.ndarray:
```

Comment:
I feel like extent is redundant.
centroids are vertices likes object.

and it will always be np.c_[self.centroids.min(axis=0), self.centroids.max(axis=0)].T 
or the same with vertices.

Reply:
The plan is to have faster lookups for the extent (or store). Beyond scope.